collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md

```diff
@@ -0,0 +1,151 @@
+* **Artifact Hub Integration:** Browse and publish Meshery Designs on Artifact Hub, expanding the reach and accessibility of reusable configurations.
+
+<a href="https://artifacthub.io" target="_blank"><img alt="Artifact Hub Kinds" src="/assets/images/posts/2024-04-09-designs-on-artifacthub/meshery-designs-on-artifact-hub.png" width="50%" align="right" /></a>
+
+Browse the [catalog](/catalog) or learn more about [how it works](https://docs.meshery.io/concepts/catalog).
+
+You can now browse Meshery Designs in the [Artifact Hub search](https://artifacthub.io/packages/search?kind=24&sort=relevance&page=1). An integration with Artifact Hub is delivered in this release, allowing you to [publish your Meshery Designs to the Artifact Hub](https://deploy-preview-1990--mesheryio-preview.netlify.app/blog/designs-on-artifact-hub), a community-driven catalog of Kubernetes applications with [over 200 Meshery designs](/blog/200-meshery-designs-on-artifacthub) to date and more on the way.
+
+### UI: A Fresh, Modern Look
+
+The Meshery UI receives a major refresh with the adoption of the [Sistent design system](https://layer5.io/projects/sistent) and Material UI v5.
+
+* **Enhanced User Experience:** Enjoy a more intuitive and visually appealing interface.
+* **New Components:** Leverage a wide range of new components from the Sistent design system.
+
+### CLI: Increased Coverage and Functionality
+
+The Meshery CLI boasts increased code coverage and a full deprecation of Apps and migration to Designs.
+
+* **Enhanced Model Support:** Dedicated commands for managing models, relationships, components, connections, credentials, and policies.
+
+### Documentation, Learning, and the Playground
+
+* **Playground Learning Paths:**  Interactive, cloud-native tutorials within Meshery Playground offer hands-on learning experiences.
+- Remote Provider: Deployment using Meshery (as a Meshery Design)
+- Remote Provider: Multi-master Postgres
+
+## Docs
+
+* [Playground] Learning Paths: Cloud Native tutorials with Meshery Playground
+  * Creating three tutorials per category, allowing learners to perform hands-on lab interactively in the live cluster environment in Meshery Playground.
+* Automated publication of End-to-End test results
+
+### Build, Test, Release: Improved Processes
+
+* **End-to-End Testing:**  Increased focus on end-to-end testing with a goal of 50% coverage.
+* **Playwright Adoption:**  Transitioning from Cypress to Playwright for more robust testing.
+* **Automated Test Results:** Stay informed with automatically published end-to-end test results.
+
+### Join the Meshery Community!
+
+We invite you to explore Meshery v0.8.0 and experience the future of cloud native management.  Join us on [Slack](https://slack.meshery.io/) to connect with the community, share your feedback, and contribute to the project.
+
+New Maintainer: [Sangram Rath](https://layer5.io/community/sangram-rath)
```

Comment:
I guess this should be https://layer5.io/community/members/sangram-rath

Thank You for the mention here.

Reply:
Good catch. Yes.

I'll add a little more context to this mention, too.